consistent-sampling/src/main/java/io/opentelemetry/contrib/samplers/ConsistentProbabilityBasedSampler.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.samplers;
+
+import io.opentelemetry.contrib.util.DefaultRandomGenerator;
+import io.opentelemetry.contrib.util.RandomGenerator;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class ConsistentProbabilityBasedSampler extends ConsistentSampler {
```

Comment:
Needs javadoc

Reply:
addressed in 9258ef4b029ba527179a7c1e6f991873849db5b5